src/hbf/scrapeZillowLinks.py

```diff
@@ -125,7 +126,7 @@ def scrapeZillowLinks(SoldHomeZillowLinks, headerInput):
                 SoldHomeHeating,
                 SoldHomeCooling,
                 SoldHomeParking,
-                SoldHomeLotSize
+                SoldHomeLotSize,
```

Comment:
why a comma after the last variable in this list? is this not going to throw an error???

Reply:
@AmritPatel see comment above

Reply:
It is part of the [style rules](https://black.readthedocs.io/en/stable/the_black_code_style.html#the-magic-trailing-comma) that are PEP8 compliant.